src/DynamoCore/Logging/DynamoAnalyticsClient.cs

```diff
@@ -232,7 +235,7 @@ public void Start()
                     RegisterADPTracker(service);
 
                 //If not ReportingAnalytics, then set the idle time as infinite so idle state is not recorded.
-                Service.StartUp(product, new UserInfo(Session.UserId), TimeSpan.FromMinutes(30));
+                Service.StartUp(product, new UserInfo(Session.UserId), hostInfo, TimeSpan.FromMinutes(30));
```

Comment:
Arguably we can add hostInfo as 4th param and make it optional but since only Dynamo will be consuming this API, I think this should be acceptable